deploy/instance/local-scripts/sync.sh

```
@@ -0,0 +1,39 @@
+zip -r "$ZIP_NAME" "$LOCAL_FOLDER"
+
+# Push file via sftp
+echo "Transferring zip file..."
+sftp "$REMOTE_HOST" << EOF
+put "$ZIP_NAME" "$REMOTE_PATH/$ZIP_NAME"
+exit
+EOF
+
+# SSH into remote, unzip file and run command
+echo "Unzipping and running command on remote..."
+ssh "$REMOTE_HOST" << EOF
```

Shellcheck (0.10.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)